src/components/shared/Extensions/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
```

Comment:
why there are have changes related with Tabs component ?

Reply:
Because it's a subtle change and I don't think so should create separate pr for it.